git-find-lfs-extensions.py

```diff
@@ -111,3 +123,10 @@ def print_line(ext, share_large, count_large, count_all, size_all, min, max):
             int(result[ext]['min']),
             int(result[ext]['max'])
         )
+
+print("\nAdd to .gitattributes:\n")
+for ext in sorted(result, key=lambda x: (result[x]['type'], x)):
+    if len(ext) > 0 and result[ext]['type'] == "binary" and result[ext]['count_large'] > 0:
+        print('*.{} filter=lfs diff=lfs merge=lfs -text'.format(
+            "".join(c if ('0' <= c <= '9') else "[" + c.upper() + c.lower() + "]" for c in ext)
+        ))
```

Comment:
I dropped off of the list because it was just too much to follow, but I'd offer the following reply to http://public-inbox.org/git/xmqqzim2vp9e.fsf@gitster.mtv.corp.google.com/

I agree with what he said, but I think because we're talking about affecting the behavior of .gitattributes that it would be better to have a distinct .gitattributes option, whether you have a similar config option as well or not.

Since .gitattributes is versioned and config options are not, I think this takes it out of the realm of personal preference. It's already too easy for somebody to screw up and not have LFS configured and re-commit binaries to a repository without being noticed until much later.

I would love a .gitconfig option I could set that would let me stop manually ignoring case in globs for git commands, but this might actually make things worse for people if it were included as a config option only.

Of course, if git supported a subset of config options you can actually version and ensure everybody else has, this wouldn't be a problem.

So, I think the correct (for today) solution is to have two options. One for .gitattributes globs and one for everything else. I could see that being controversial so I think it's worth raising on the list. If you don't want to do it on my behalf, I'll try to figure out how to make one of the options for replying to it work (they seem nontrivial, correct me if I'm wrong!).